packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -185,7 +170,7 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
    * @return The median exchange rate for `token`.
    */
   function medianRate(address token) external view returns (uint256, uint256) {
-    return (rates[token].getMedianValue(), numRates(token) == 0 ? 0 : DENOMINATOR);
+    return (rates[token].getMedianValue(), numRates(token) == 0 ? 0 : FIXED1_UINT);
```

Comment:
Add issue to, once #2890 is merged, make this return just a fixidity fraction.

Reply:
https://github.com/celo-org/celo-monorepo/issues/2929

Reply:
Left as-is for now due to needed celo-blockchain changes.